tests/common.py

```diff
@@ -62,3 +62,13 @@ def vllm_setup_test(runner, args, mock_popen, *_mock_args):
 def assert_tps(args, tps):
     assert args[-2] == "--tensor-parallel-size"
     assert args[-1] == tps
+
+
+@mock.patch("instructlab.model.backends.vllm.check_api_base", return_value=False)
```

Comment:
This change should be reverted. Just use the regular `vllm_setup_test` helper unless you test the serve/vllm loop specifically. Otherwise you affect `ragas` analyticsBroker thread pulled by their import. (I think the design of the ragas library could be of course better - not spawning threads on import would be nice - but we have to live with it.)